packages/runtime/container-runtime/src/opLifecycle/definitions.ts

```diff
@@ -34,6 +34,13 @@ export interface IBatch {
 	 * The reference sequence number for the batch
 	 */
 	readonly referenceSequenceNumber: number | undefined;
+	/**
+	 * Wether or not the batch contains at least one op which was produced as the result
+	 * of processing another op. This means that the batch must be rebased before
+	 * submitted, to ensure that all ops have the same sequence numbers and a
```

Comment:
maybe worth elaborating a bit more why it's a problem / why solution is to have same ref seq number (you actually miss "ref" word here, making it harder for uneducated developer to follow the logic).